features/providers/check_your_answers.feature

```diff
@@ -161,24 +158,22 @@ Feature: Checking answers backwards and forwards
       And the answer for 'Own home' should be 'No'
 
     @javascript @vcr
-    Scenario: I want to remove savings via the capital check your answers page
+    Scenario: I want to view savings via the capital check your answers page
       Given I complete the passported journey as far as capital check your answers
       And I click Check Your Answers Change link for 'Savings and investments'
       Then I should be on a page showing 'Does your client have any savings and investments?'
-      Then I deselect 'Cash savings'
+      Then I click 'Continue'
```

Comment:
That a particular saving can be deselected via a page entry is a test of that page and is done via a request spec. What needs to be tested here is the flow behaviour - that is, that a user can get to this page from check your answers and return successfully.